### PR TITLE
Php upgrade

### PR DESCRIPTION
**Pull request by sven7777**

### CODE REVIEW

File: src/RoutingApp.php [hunk 1]
```diff
@@ -43,9 +40,9 @@ public function call(Context $context) {
 		// Force pages to their designated protocol if specified
 		if ($route->getProtocol() != null) {
 			if ($currentProtocol != $route->getProtocol()) {
-				$url = Url::fromString(sprintf("%s://%s/", $route->getProtocol(), $currentHost))
-						->join($currentUrl)
-						->addParams($_GET);
+				/** @var Url $url */
+				$url = Url::fromString(sprintf("%s://%s%s", $route->getProtocol(), $currentHost, $currentUrl));
```
Comment (oh4real) [inline src/RoutingApp.php]: Add Params is removed?

File: test/DefaultInvokerTest.php [hunk 2]
```diff
@@ -62,7 +78,34 @@ public function setup() {
...
+				->command('Fliglio\Routing.StubResource.addFoo')
+				->method(Http::METHOD_POST)
+				->build()
+		);
+	}
+
+	/** @expectedException Fliglio\Flfc\Exceptions\RedirectException */
+	public function testChangeOfProtocol() {
+		// given
+		$this->request->setUrl('/biz/123');
```
Comment (oh4real) [inline test/DefaultInvokerTest.php]: Thanks for adding test. Add request with GET params?

File: src/RoutingApp.php (hunk 1 above)
Comment (sven7777) [reply src/RoutingApp.php]: This whole thing would have never worked had it actually ever been hit. GET params are empty at this point. Current url has the query params.

File: test/DefaultInvokerTest.php (hunk 2 above)
Comment (sven7777) [reply test/DefaultInvokerTest.php]: Added a second test. One with and one without.

File: src/RoutingApp.php [hunk 3]
```diff
@@ -43,9 +40,12 @@ public function call(Context $context) {
 		// Force pages to their designated protocol if specified
 		if ($route->getProtocol() != null) {
 			if ($currentProtocol != $route->getProtocol()) {
-				$url = Url::fromString(sprintf("%s://%s/", $route->getProtocol(), $currentHost))
-						->join($currentUrl)
-						->addParams($_GET);
+				$getParams = implode("&", $_GET);
```
Comment (oh4real) [inline src/RoutingApp.php]: $getParams extracted. I don't see them used?

Comment (sven7777) [reply src/RoutingApp.php]: Whoops, left that in from testing to see how the GET params were being used and sent in. Remove it.

**Pull request merged**